db/data/20201112075313_remove_unassociated_facilities_from_production.rb

```diff
@@ -0,0 +1,12 @@
+class RemoveUnassociatedFacilitiesFromProduction < ActiveRecord::Migration[5.2]
+  def up
+    return if SIMPLE_SERVER_ENV != "production"
+    return if ENV["DEFAULT_COUNTRY"] != "IN"
+
+    Facility.where(facility_group_id: nil).delete_all
```

Comment:
Do we need to `discard_data` instead? Are we sure there's no data attached to these facilities?

Reply:
Yes, [checked](https://simpledotorg.slack.com/archives/GJ2AUUASK/p1605075304000700). Praveen also checked with the CVHOs to whom these facilities may belong to.

These are truly bad facilities, unlike, say PATH, who served some purpose in the past.